target_redshift/db_sync.py

```diff
@@ -12,12 +12,18 @@
 import sys
 import time
 from datetime import datetime
+from functools import partial
```

Comment:
The main point of this is I noticed a lot of duplication in the original target
instead of
```
cur.execute(sql)
self.logger(blah)
```

I figure I could just inline it  fairly simply

```
def execute_sql(cursor,logger,sql):
...

run = partial(execute_sql, cur, self.logger)
run (sql)
```

Hopefully that makes sense